teams/README.md

```diff
@@ -0,0 +1,80 @@
+# Teams
+
+This folder holds all teams in the TiDB projects. This document defines governance policies of the teams.
+
+Teams are persistent open groups that focus on a part of the TiDB projects. A team has its reviewer, committer and maintainer, and owns one or more repositories.Team level decision making comes from its maintainers.
+
+The [Technical Oversight Committee (TOC)](/toc/README.md) is in charge of team creation and retirement, and arbitrates divergence between team. But TOC members don't participant team level decision making directly.
+
+## Roles and Responsibilities
+
+### Reviewers
+
+Reviewers are individuals who actively make contribution and are willing to participate in the code review of new contributions. We identify reviewers from active contributors. The committers should explicitly solicit reviews from reviewers. High-quality code reviews prevent technical debt for long-term and are crucial to the success of the project. A pull request to the project has to be reviewed by at least one reviewer in order to be merged.
+
+Review access is by invitation only and must be approved by consensus of the maintainers. A reviewer is considered emeritus by their own declaration. An emeritus reviewer may request reinstatement of review access from the maintainers, which will be sufficient to restore him or her to active reviewer status.
+
+Review access can be revoked by consensus by the maintainers (except the reviewer in question if they are also a maintainer).
```

Comment:
I'm curious under what circumstances a reviewer is also a maintainer?

Reply:
A maintainer is a reviewer effectively, but you're right we can remove this sentence to reduce confusion. Reviewer in our context is reviewer has _only_ reviewer permissions.